src/solver/simulation/solver.hxx

```diff
@@ -231,10 +238,12 @@ private:
             // 9 - Write results for the current year
             if (yearByYear)
             {
+                TimeElapsed::Timer timer("Year-by-year export", "yby_export", false, pTimeElapsedAggregator);
```

Comment:
Why not print **Year-by-year export** this into the logs ?
If we eventually do print **Year-by-year export** into logs, we should consider removing the **Timer**'s **verbose** member.

Reply:
We may not want to display too many things in the logs.

It would be best to replace the current
```
<Operation> starting....

<Operation> ended
```
by
```
<Operation> starting...

<Operation> ended in 43ms
```